exercises/meetup/example.js

```diff
@@ -56,15 +53,17 @@ function _getDayIndex(day) {
     'saturday': 6
   };
 
-  day = day.toLowerCase();
+  var temp = day.toLowerCase();
 
-  return daysInd[day];
+  return daysInd[temp];
 }
 
 function _find(ary, callback) {
   for (var i = 0; i < ary.length; i++) {
     if (callback(ary[i], i, ary)) { return ary[i]; }
   }
+  return null;
```

Comment:
`return undefined` is implicit, and preferred by the [style guide](https://github.com/airbnb/javascript#arrows--implicit-return), so let's not do this.